runtime/stateManager.py

```diff
@@ -38,6 +39,9 @@ def make_subscription_map():
         return subscription_map
 
     def make_command_map(self):
+        """
+        Provides mapping between command input and the state manager behavior.
```

Comment:
Perhaps something more like "Create a mapping between StateManager commands and SM behavior"